config/params.php

```diff
@@ -91,9 +91,9 @@
     ],
     'yiisoft/yii-console' => [
         'commands' => [
-            DebugResetCommand::COMMAND_NAME => DebugResetCommand::class,
-            DebugContainerCommand::COMMAND_NAME => DebugContainerCommand::class,
-            DebugEventsCommand::COMMAND_NAME => DebugEventsCommand::class,
+            'debug:reset' => DebugResetCommand::class,
```

Comment:
Let's leave it as it is

Reply:
What is better, strings or constant? And why?
We should use one way in all packages.

Reply:
@xepozz command name is useless with attributes and without attributes Symfony console raises deprecation warning.

Reply:
I like constants because they already exist in console command classes and it's easy to reference to the names rather than writing the name again and keep them consistent